config/locales/en.yml

```diff
@@ -95,6 +95,39 @@ en:
     shared:
       label: 'Label'
       data_type: 'Field type'
+      name: 'Name'
+      required: 'Required'
+    forms:
+      name:
+        help:
```

Comment:
@tara-harwood a campaigner can click to see help when filling in the field's name attribute. Below is copy. Let me know if it's ok.